charmcraft/commands/pack.py

```diff
@@ -183,19 +174,28 @@ def _pack_bundle(self):
                 "file {!r}.".format(str(bundle_filepath))
```

Comment:
Could we link to a bundle spec in the error message to make this easier to solve?

Reply:
sounds like a sane idea, but the bundle reference https://juju.is/docs/sdk/bundle-reference looks more like a mix of mini-how-to's today. This might be catered to a previous bundle world as I don't see the name mentioned anywhere. Should we fix that first?

Reply:
Yeh, I think that bundle reference is a little out of date. I don't know what the defacto source of info on bundle spec is now? @jameinel?

Reply:
We can add the link in a follow-up PR.